lib/OpenQA/Worker/WebUIConnection.pm

```diff
@@ -194,23 +194,23 @@ sub _setup_websocket_connection {
                     $command_handler->handle_command(@_);
                 });
             $tx->on(
-                finish => sub {
+                finish => sub ($tx, $code, $reason = 'no reason') {
                     # uncoverable subroutine
                     # https://progress.opensuse.org/issues/55364
-                    my (undef, $code, $reason) = @_;
 
                     # Subprocesses reset the event loop (which triggers this event),
                     # and since only the main worker process uses the WebSocket we
                     # can safely ignore this (and do not want to reconnect)
+                    # uncoverable statement
                     return unless $websocket_pid == $$;
 
                     # ignore if the connection was disabled from our side
+                    # uncoverable statement
                     return log_debug("Websocket connection to $websocket_url finished from our side.")
                       unless $self->websocket_connection;
 
-                    $reason //= 'no reason';
-                    $self->websocket_connection(undef);
-                    $self->_set_status(
+                    $self->websocket_connection(undef);    # uncoverable statement
+                    $self->_set_status(                    # uncoverable statement
```

Comment:
It seems `websocket_connection` is chainable. So `$self->websocket_connection(undef)->_set_status(` would get rid of another line.

Reply:
done